applications/nrf_desktop/src/modules/hid_forward.c

```diff
@@ -275,8 +309,18 @@ static void hidc_ready(struct bt_gatt_hids_c *hids_c)
 				LOG_ERR("Cannot subscribe to report (err:%d)",
 					err);
 			} else {
-				LOG_INF("Subscriber to rep id:%d",
-					bt_gatt_hids_c_rep_id(rep));
+				u8_t report_id = bt_gatt_hids_c_rep_id(rep);
+
+				LOG_INF("Subscriber to rep id:%d", report_id);
+				if (IS_ENABLED(CONFIG_DESKTOP_HID_FORWARD_LOW_LATENCY_ON_HID_INPUT)) {
+					/* Application assumes that only one
+					 * subscriber provides HID input report
+					 * with given ID.
+					 */
+					__ASSERT_NO_MSG(rep_id_2_sub_idx[report_id] ==
```

Comment:
I don't think we should make such assumption.

Reply:
discussed offline